source/working-with-data.txt

```diff
@@ -9,16 +9,16 @@ Working With Data
 .. toctree::
   :titlesonly:
 
-  reference/crud
-  reference/queries
-  reference/text-search
-  reference/aggregation
-  reference/map-reduce
-  reference/persistence-configuration
-  reference/nested-attributes
-  reference/callbacks
-  reference/sessions
-  reference/transactions
+  CRUD Operations <reference/crud>
+  Queries <reference/queries>
+  Text Search <reference/text-search>
+  Aggregation Pipeline <reference/aggregation>
+  Map/Reduce <reference/map-reduce>
```

Comment:
The Style Guide generally discourages slashes, and MapReduce appears to be the preferred term:

https://en.wikipedia.org/wiki/MapReduce